walletnode/services/artworksearch/helper_test.go

```diff
@@ -0,0 +1,79 @@
+package artworksearch
+
+import (
+	b64 "encoding/base64"
```

Comment:
base64 is conflicted with other packages of same name, if not, i think it's better not  rename the package name

Reply:
@alonlong agree. but  this is how its done throughout in existing code I'm just keeping things consistent.